### ISSUE TITLE
[Bug]: Needs ARIA , title, target, rel attributes to Social Links and Form Sections along with Missing alt attributes in Image Tags

### ISSUE THREAD

**Issue by NarendraKoya999** [AbhiPatel10/AISKCON-CONSTRUCTION#498]
### Issue Description:
- We need to add `aria-label OR aria-labelby attributes to Form Section`
- Also, add `aria-label, title, target, rel attributes to Respective Social Icons in All Web Pages`
- Missing `alt attributes to Image Tags in ServicesSection/index.html  File`

### Purpose of Raising this Issue:
- The `aria-label attribute` provides a way to place a descriptive text label on an object, such as a link, when there are no elements visible on the page that describe the object. If descriptive elements are visible on the page, the aria-labelledby attribute should be used instead of aria-label .
- The `target attribute` inside anchor tags ( <a> ) tells the browser where the linked document should be loaded. It's optional, and defaults to _self when no value is provided.
- The` rel attribute` specifies the relationship between the current document and the linked document. Only used if the href attribute is present.
- The `title attribute` gives users an important indication as to where they will be directed to when they click on a link. If the user hovers with the mouse over a link reference, the title text, which was defined in the attribute, will be displayed.
- The `required alt attribute` specifies an alternate text for an image, if the image cannot be displayed. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of slow connection, an error in the src attribute, or if the user uses a screen reader).

### For Reviewers, I want to work on this Issue

**Comment by NarendraKoya999** [comment]
Hi @Neilblaze . Kindly Review on this Issue as soon as Possible.

**Comment by NarendraKoya999** [comment]
Hi @AbhiPatel10 . Please Review on this Issue. Thank You !
